cmake/config.cmake

```diff
@@ -145,6 +145,8 @@ set(USE_RANDOM OFF)
 # Whether use NNPack
 set(USE_NNPACK OFF)
 
+set(USE_TFLITE ON)
```

Comment:
We may want to leave that option off by default?

Reply:
nope

Reply:
+1, since it is an optional contrib feature, we want to leave it off by default